visualdl/server/graph.py

```diff
@@ -483,23 +486,7 @@ def add_edge(self, source, target, label, **kwargs):
 
 def draw_graph(model_pb_path, image_dir):
```

Comment:
The function name doesn't apply anymore. Also the we no longer need image_dir.

Reply:
Yes. This will also be renamed in the cleanup PR.